lib/util.js

```diff
@@ -767,7 +787,14 @@ module.exports = {
         if (this.getHeaderFamily(contentType) === HEADER_TYPE.XML) {
           schemaType = SCHEMA_FORMATS.XML;
         }
-        bodyData = safeSchemaFaker(bodyObj.schema || {}, bodyTypeOption, this.components, schemaType, indentCharacter);
+        if (requestType === REQUEST_TYPE.ROOT) {
+          resolveTo = this.options.requestParametersResolution;
+        }
+        else if (requestType === REQUEST_TYPE.EXAMPLE) {
```

Comment:
else?
is there a default? what happens if I send something other than root/example to the function?

Reply:
The default value of `resolveTo` is an empty string which means if I send something other than root/example then the `JSON-schema-faker` will fake the schema properties instead of using examples.

Reply:
the default value of resolveTo should be `schema` or `example`. Why do we have a blank string?